balor/balor.ino

```diff
@@ -18,16 +18,19 @@
 #include <ros.h>
 #include <std_msgs/Header.h>
 #include <std_msgs/Empty.h>
-#include <std_msgs/Int32.h>
+#include <std_msgs/String.h>
 #include <ackermann_msgs/AckermannDrive.h>
 
+// C++ includes
+#include <math.h>
+
 ros::NodeHandle ros_nh;
 
 std_msgs::Header heartbeat_msg;
 ros::Publisher heartbeat_pub("heartbeat", &heartbeat_msg);
 
-std_msgs::Int32 param_msg;
-ros::Publisher param_pub("param", &param_msg);
+std_msgs::String error_msg;
+ros::Publisher error_pub("platform/debug/error", &error_msg);
 
 ackermann_msgs::AckermannDrive debug_drive_msg;
 ros::Publisher debug_drive_pub("/platform/debug/drive", &debug_drive_msg);
```

Comment:
I think the leading `/` needs to be removed here

Reply:
Removed